libpkg/scripts.c

```diff
@@ -198,6 +198,9 @@ pkg_script_run(struct pkg * const pkg, pkg_script type)
 			}
 
 			if (WEXITSTATUS(pstat) != 0) {
+				if (WEXITSTATUS(pstat) == 3)
```

Comment:
Opps, I meant changing it to:

`if (WEXITSTATUS(pstat) != 0 && WEXITSTATUS(pstat) != 3)`